### PR TITLE
/doc/tidb-cloud add tpcc 7.1 test result

### PR DESCRIPTION
**Pull request by shaoxiqian**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add TiDB Cloud TPC-C Performance Test Report for v7.1.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: media/tidb-cloud/privatelink.png
Comment (Oreoxmt) [inline media/tidb-cloud/privatelink.png:1]: Please remove `media/tidb-cloud/privatelink.png`.

To reduce the maintenance cost of images, it is not recommended to use TiDB Cloud console screenshots. You can directly link to the corresponding user documentation instead, such as https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections.

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:6]:
```suggestion

This document provides the TPC-C performance test result for the TiDB Dedicated cluster with a cluster version of v7.1.0. This report can be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.

```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   ```
+   set tidb_build_stats_concurrency=16;
+   set tidb_distsql_scan_concurrency=16;
+   set tidb_index_serial_scan_concurrency=16;
+   ```
+5. Run the command to perform stress tests on TiDB via privatelink. For each concurrency, the test takes 2 hours.
+   ```
+   go-tpc tpcc --host {{privatelink}} -P 4000 --warehouses 1000 run -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
+   ``` 
+6. Extract the tpmC data of New Order from the result.
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:74]:
````suggestion
## Test environment

- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
- Cluster version: v7.1.0
- Cloud provider: AWS
- Cluster configuration:

   | Node type | Node size | Node quantity | Node storage |
   |:----------|:----------|:----------|:----------|
   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
   | TiKV      | 16 vCPU, 64 GiB | 3 | 100 GiB |

## Test steps

This section introduces how to perform the TPC-C performance test step by step.

1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#test-environment) requirements.

   For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).

2. Connect to the newly created cluster and create a database named `tpcc`.

   To connect to the cluster, see [Connect via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md).

   To create the `tpcc` database, execute the following SQL statement:

   ```sql
   CREATE DATABASE tpcc;
   ```

3. Load TPC-C data to the `tpcc` database.

   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:

      ```shell
      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
      ```

   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:

      ```shell
      go-tpc tpcc --host {{privatelink}} --warehouses 1000 prepare -P 4000 -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
      ```

4. Before conducting the TPC-C test, collect the following statistics:

   ```sql
   ANALYZE TABLE customer;
   ANALYZE TABLE district;
   ANALYZE TABLE history;
   ANALYZE TABLE item;
   ANALYZE TABLE new_order;
   ANALYZE TABLE order_line;
   ANALYZE TABLE orders;
   ANALYZE TABLE stock;
   ANALYZE TABLE warehouse;
   ```

   To accelerate the collection of statistics, execute the following SQL statements before collecting:

   ```sql
   SET tidb_build_stats_concurrency=16;
   SET tidb_distsql_scan_concurrency=16;
   SET tidb_index_serial_scan_concurrency=16;
   ```

5. Run the following `go-tpc tpcc` command to conduct stress tests on the TiDB Dedicated cluster. For each concurrency, the test takes two hours.

   ```shell
   go-tpc tpcc --host {{privatelink}} -P 4000 --warehouses 1000 run -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
   ```

6. Extract the tpmC data of New Order from the result.

   TPC-C uses tpmC (transactions per minute) to measure the maximum qualified throughput (MQTh, Max Qualified Throughput). The transactions are the NewOrder transactions and the final unit of measure is the number of new orders processed per minute.
````

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+The TPC-C performance of v7.1.0 is 
+
+| Threads |  v7.1.0 tpmC |
+|:----------|:----------|
+|50|37904|
+|100|67790|
+|200|93479|
+
+![TPC-C](/media/tidb-cloud/v7.1.0-tpmC.png)
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:86]:
```suggestion
The TPC-C performance of v7.1.0 is as follows:

| Threads |  v7.1.0 tpmC |
|:--------|:----------|
| 50  | 37,904 |
| 100 | 67,790 |
| 200 | 93,479 |

![TPC-C](/media/tidb-cloud/v7.1.0-tpmC.png)
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+```yaml
+[raft-engine]
+prefill-for-recycle = true
+```
+
+### Prepare test data and run tpcc
+
+1. Deploy TiDB v7.1.0 using TiDB Cloud.
+2. Connect the cluster and create a database named `tpcc`: 
+   ![privatelink](/media/tidb-cloud/privatelink.png)
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:43]: Is it necessary for users to use a [private endpoint]((https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections)) when connecting to TiDB Cloud and executing the `go-tpc tpcc` command? Can they use the standard connection or VPC peering instead?

Comment (Yui-Song) [reply tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:43]: For performance testing, the private endpoint is recommended.

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   ```
+   set tidb_build_stats_concurrency=16;
+   set tidb_distsql_scan_concurrency=16;
+   set tidb_index_serial_scan_concurrency=16;
+   ```
+5. Run the command to perform stress tests on TiDB via  Private Endpoint. For each concurrency, the test takes 2 hours.
+   ```
+   go-tpc tpcc --host {{privatelink}} -P 4000 --warehouses 1000 run -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
+   ``` 
+6. Extract the tpmC data of New Order from the result.
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:69]:
```suggestion
## Test environment

- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
- Cluster version: v7.1.0
- Cloud provider: AWS
- Cluster configuration:

   | Node type | Node size | Node quantity | Node storage |
   |:----------|:----------|:----------|:----------|
   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
   | TiKV      | 16 vCPU, 64 GiB | 3 | 100 GiB |

## Test steps

This section introduces how to perform the TPC-C performance test step by step.

1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#test-environment) requirements.

   For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).

2. Connect to the newly created cluster and create a database named `tpcc`.

   To connect to the cluster, see [Connect via Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md).

   To create the `tpcc` database, execute the following SQL statement:

   ```sql
   CREATE DATABASE tpcc;
   ```

3. Load TPC-C data to the `tpcc` database.

   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:

      ```shell
      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
      ```

   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:

      ```shell
      go-tpc tpcc --host {{privatelink}} --warehouses 1000 prepare -P 4000 -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
      ```

4. Before conducting the TPC-C test, collect the following statistics:

   ```sql
   ANALYZE TABLE customer;
   ANALYZE TABLE district;
   ANALYZE TABLE history;
   ANALYZE TABLE item;
   ANALYZE TABLE new_order;
   ANALYZE TABLE order_line;
   ANALYZE TABLE orders;
   ANALYZE TABLE stock;
   ANALYZE TABLE warehouse;
   ```

   To accelerate the collection of statistics, execute the following SQL statements before collecting:

   ```sql
   SET tidb_build_stats_concurrency=16;
   SET tidb_distsql_scan_concurrency=16;
   SET tidb_index_serial_scan_concurrency=16;
   ```

5. Run the following `go-tpc tpcc` command to conduct stress tests on the TiDB Dedicated cluster. For each concurrency, the test takes two hours.

   ```shell
   go-tpc tpcc --host {{privatelink}} -P 4000 --warehouses 1000 run -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
   ```

6. Extract the tpmC data of New Order from the result.

   TPC-C uses tpmC (transactions per minute) to measure the maximum qualified throughput (MQTh, Max Qualified Throughput). The transactions are the NewOrder transactions and the final unit of measure is the number of new orders processed per minute.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Test result
+
+The TPC-C performance of v7.1.0 is 
+
+| Threads |  v7.1.0 tpmC |
+|:----------|:----------|
+|50|36159|
+|100|66742|
+|200|94406|
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:79]:
```suggestion
The TPC-C performance of v7.1.0 is as follows:

| Threads |  v7.1.0 tpmC |
|:--------|:----------|
| 50  | 36,159 |
| 100 | 66,742 |
| 200 | 94,406 |
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## Test overview
+
+This test aims at showing the TPC-C performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
+- Cluster version: v7.1.0
+- Cloud provider: AWS
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:18]:
```suggestion
- Cloud provider: AWS (us-west-2)
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 100 GiB |
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:24]:
```suggestion
   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+summary: Introduce the TPC-C performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+This document provides the TPC-C performance test result for the TiDB Dedicated cluster with a cluster version of v7.1.0. This report can be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:8]:
```suggestion
This document provides the TPC-C performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
```

Comment (qiancai) [reply tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:8]: TiDB version is used on UI
image

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:
+
+      ```shell
+      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
+      ```
+
+   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:
+
+      ```shell
+      go-tpc tpcc --host {{privatelink}} --warehouses 1000 prepare -P 4000 -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:55]: Do we need to provide the range or a typical value of thread to achieve the test result?

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+      ```
+
+4. Before conducting the TPC-C test, collect the following statistics:
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:58]:
```suggestion
4. To ensure that the TiDB optimizer can generate the optimal execution plan, execute the following SQL statements to collect statistics before conducting the TPC-C test:
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+   SET tidb_index_serial_scan_concurrency=16;
+   ```
+
+5. Run the following `go-tpc tpcc` command to conduct stress tests on the TiDB Dedicated cluster. For each concurrency, the test takes two hours.
+
+   ```shell
+   go-tpc tpcc --host {{privatelink}} -P 4000 --warehouses 1000 run -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
+   ```
+
+6. Extract the tpmC data of New Order from the result.
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:86]:
```suggestion
6. Extract the tpmC data of `new_order` from the result.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+
+   TPC-C uses tpmC (transactions per minute) to measure the maximum qualified throughput (MQTh, Max Qualified Throughput). The transactions are the NewOrder transactions and the final unit of measure is the number of new orders processed per minute.
+
+## Test result
+
+The TPC-C performance of v7.1.0 is as follows:
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:92]:
```suggestion
The TPC-C performance of v7.1.0 in the [test environment] is as follows:
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:
+
+      ```shell
+      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
+      ```
+
+   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:
+
+      ```shell
+      go-tpc tpcc --host {{privatelink}} --warehouses 1000 prepare -P 4000 -D tpcc -T {{thread}} --time 2h0m0s -p {{password}} --ignore-error
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:55]:
```suggestion
      go-tpc tpcc --host ${HOST} --warehouses 1000 prepare -P 4000 -D tpcc -T ${THREAD} --time 2h0m0s -p ${PASSWORD} --ignore-error
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+
+3. Load TPC-C data to the `tpcc` database.
+
+   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:
+
+      ```shell
+      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
+      ```
+
+   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database:
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:52]:
```suggestion
   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database. Replace `${HOST}`, `${THREAD}`, and `${PASSWORD}` with your actual values. This document conducts three tests with `${THREAD}` value of `50`, `100`, and `200`.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+
+## Test steps
+
+This section introduces how to perform the TPC-C performance test step by step.
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#test-environment) requirements.
+
+   For more information, see [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
+
+2. Connect to the newly created cluster and create a database named `tpcc`.
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:34]:
```suggestion
2. On the benchmark executor, connect to the newly created cluster and create a database named `tpcc`.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 17]
```diff
@@ -0,0 +1,100 @@
...
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+## Test steps
+
+This section introduces how to perform the TPC-C performance test step by step.
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#test-environment) requirements.
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:30]:
```suggestion
1. In the [TiDB Cloud console](https://tidbcloud.com/), create a TiDB Dedicated cluster that meets the [test environment](#tidb-cluster) requirements.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+
+This test aims at showing the TPC-C performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated)
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:20]:
```suggestion
- Cluster type: [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier)
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+summary: Introduce the TPC-C performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+This document provides the TPC-C performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:8]:
```suggestion
This document provides the TPC-C performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB v7.1.0 clusters.
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+
+   ```shell
+   go-tpc tpcc --host ${HOST} -P 4000 --warehouses 1000 run -D tpcc -T ${THREAD} --time 2h0m0s -p ${PASSWORD} --ignore-error
+   ```
+
+6. Extract the tpmC data of `NEW_ORDER` from the result.
+
+   TPC-C uses tpmC (transactions per minute) to measure the maximum qualified throughput (MQTh, Max Qualified Throughput). The transactions are the NewOrder transactions and the final unit of measure is the number of new orders processed per minute.
+
+## Test result
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:101]:
```suggestion
## Test results
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 21]
```diff
@@ -0,0 +1,111 @@
...
+
+3. Load TPC-C data to the `tpcc` database.
+
+   1. The test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following command:
+
+      ```shell
+      curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/pingcap/go-tpc/master/install.sh | sh
+      ```
+
+   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database. Replace `${HOST}`, `${THREAD}`, and `${PASSWORD}` with your actual values. This document conducts three tests with `${THREAD}` value of `50`, `100`, and `200`.
```
Comment (qiancai) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:63]:
```suggestion
   2. Run the following `go-tpc tpcc` command to import 1,000 warehouses to the `tpcc` database. Replace `${HOST}`, `${THREAD}`, and `${PASSWORD}` with your actual values. This document conducts three tests with the `${THREAD}` value of `50`, `100`, and `200`.
```

**Review by qiancai — APPROVED** [review]
other LGTM

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+### Benchmark executor
```
Comment (Yui-Song) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:30]: Please also add the AWS region for the Benchmark executor.

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md [hunk 23]
```diff
@@ -0,0 +1,111 @@
...
+
+The benchmark executor sends SQL queries to the TiDB cluster. In this test, its hardware configuration is as follows:
+
+- Machine type: AWS EC2
```
Comment (Oreoxmt) [inline tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:34]:
```suggestion
- Machine type: Amazon EC2 (us-west-2)
```

File: tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md (hunk 22 above)
Comment (Oreoxmt) [reply tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md:30]: Done https://github.com/pingcap/docs/pull/13696/commits/5afd391e501e127a52a415db6fad6738291c8a7d

**Review by Oreoxmt — APPROVED** [review]

**Review by Yui-Song — COMMENTED** [review]
Please add labels for the horizontal and vertical axes.

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**